requests/models.py

```diff
@@ -577,6 +577,10 @@ def prepare_hooks(self, hooks):
         for event in hooks:
             self.register_hook(event, hooks[event])
 
+    def send(self, session):
```

Comment:
"Harmless" seems in conflict with "Extra surface area that doesn't seem to provide much benefit"